[Bug Report] Wrong path to jest binary in `debug:test` script of vuetify's package.json

### Versions and Environment
**Vuetify:** 2.0.0-beta.0
**Vue:** 2.6.4
**Browsers:** Chrome 72.0.3626.121
**OS:** Windows 10

### Steps to reproduce
From `vuetify` package running `npm run debug:test` and opening chrome inspector, it turns into error:

```
internal/modules/cjs/loader.js:584
    throw err;
    ^

Error: Cannot find module 'C:\Users\beat.grabe\Beat Projects\vuetify\packages\vuetify\node_modules\jest\bin\jest.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:582:15)
    at Function.Module._load (internal/modules/cjs/loader.js:508:25)
    at Function.Module.runMain (internal/modules/cjs/loader.js:754:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)
```

### Expected Behavior
The `debug:test` script to run

### Actual Behavior
Error occurs

### Reproduction Link
<a href="https://github.com/beatgrabe/vuetify/tree/dev/packages/vuetify" target="_blank">https://github.com/beatgrabe/vuetify/tree/dev/packages/vuetify</a>

### Other comments
I guess this error was introduced by moving into lerna mono repo and forgetting about adapting the path to the jest binary, which is now part of that upper `package.json` dependency…


<!-- generated by vuetify-issue-helper. DO NOT REMOVE -->

--- a/packages/vuetify/package.json
+++ b/packages/vuetify/package.json
@@ -33,7 +33,7 @@
     "build:dist": "rimraf dist && cross-env NODE_ENV=production webpack --config build/config.js --progress --hide-modules",
     "tsc": "tsc",
     "build:lib": "node build/build-lib.js",
-    "debug:test": "cross-env NODE_ENV=test node --inspect --inspect-brk ./node_modules/jest/bin/jest.js --no-cache -i --verbose",
+    "debug:test": "cross-env NODE_ENV=test node --inspect --inspect-brk ../../node_modules/jest/bin/jest.js --no-cache -i --verbose",
     "test": "node build/run-tests.js",
     "test:unix": "cross-env NODE_ENV=test jest",
     "test:win32": "cross-env NODE_ENV=test jest -i",

